packages/react-slider/src/utils/calculateSteps.ts

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { clamp } from './clamp';
+
+/**
+ * Calculates the `step` position based off of a `Mouse` or `Touch` event relative to the size of the rail.
+ */
+export const calculateSteps = (
+  ev: React.PointerEvent<HTMLDivElement>,
+  railRef: React.RefObject<HTMLDivElement>,
+  min: number,
+  max: number,
+  step: number,
+  vertical: boolean,
+  dir: 'ltr' | 'rtl',
+): number => {
+  const currentBounds = railRef?.current?.getBoundingClientRect();
+  const sliderSize = vertical ? currentBounds!.height : currentBounds!.width;
```

Comment:
Do you know for sure that `currentBounds` will always be defined? It might be good to have a fallback if it isn't.